torch/distributed/rpc/backend_registry.py

```diff
@@ -76,7 +77,7 @@ def process_group_init_backend_handler(
                 )
             )
         # TODO: add try-except and destroy _agent in all processes if any fails.
-        return dist.ProcessGroupAgent(self_name, group, num_send_recv_threads, kwargs["rpc_timeout"])
+        return dist.ProcessGroupAgent(self_name, group, num_send_recv_threads, rpc_timeout)
```

Comment:
```cpp
return dist.ProcessGroupAgent(self_name, group, kwargs["num_send_recv_threads"], rpc_timeout)
```

Reply:
This reads a bit odd as we are passing args from `kwargs` before regular arguments. If we want to do this, then we should change the argument order in process group agent?